src/components/CreditCard/CreditCard.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './CreditCard.css';
+
+function CreditCard(props) {
```

Comment:
Well... I can see that you didn't waist much time with the styling xD, but what matters is that the info is all there and looks fine!